beacon-chain/rpc/prysm/v1alpha1/node/server.go

```diff
@@ -227,7 +227,11 @@ func (ns *Server) GetETH1ConnectionStatus(ctx context.Context, _ *empty.Empty) (
 	errs := ns.POWChainInfoFetcher.ETH1ConnectionErrors()
 	// Extract string version of the errors.
 	for _, err := range errs {
-		errStrs = append(errStrs, err.Error())
+		if err == nil {
+			errStrs = append(errStrs, "")
```

Comment:
What do we need an empty string for? Why not just skip this item?

Reply:
If we leave the empty string out, it might be confusing in the case that some of the connections have errors and others do not. For example, let's say we have connections `conn1, conn2, conn3` and errors only on the first and third connection. Then the error strings could either be `err1, "", err2` or `err1 err2` (if we don't use the empty strings). It seems like there might be some ambiguity in the second case, because `err2` could either map to `conn2` or `conn3`. WDYT?

Reply:
This makes perfect sense given that `ETH1ConnectionErrors` returns an error for each connection. However the documentation says

>ETH1ConnectionErrors returns a slice of errors for each HTTP endpoint. An error of nil means the connection was successful.

Calling `Error()` on a nil error panics: https://goplay.tools/snippet/UdO-F9otnKc

Reply:
This makes perfect sense given that `ETH1ConnectionErrors` returns a value for each connection. Thanks!